src/operator/LimitsField.tsx

```diff
@@ -36,12 +40,35 @@ const LimitsField: FC<Props> = ({type, name, limits, onChangeLimits}) => {
     onChangeLimits(newLimits)
   }
 
+  if (type == InputType.Number) {
+    return hasFocus ? (
+      <Input
+        type={type}
+        name={name}
+        value={value}
+        onChange={onChange}
+        onBlur={() => setHasFocus(false)}
+        testID={`limits-${name}--input`}
+      />
+    ) : (
+      <Input
+        type={InputType.Text}
+        name={name}
+        value={csv}
+        onChange={onChange}
+        onFocus={() => setHasFocus(true)}
+        testID={`limits-${name}--input`}
+      />
+    )
+  }
```

Comment:
I'm a little bit confused about the intent here. If I'm reading it correctly, it looks like the top level `if` checks if the type is a number. If it is, it checks if the component has focus. If it doesn't, it drops down into the bottom case, and sets the type to be `text`. Is this intended?